src/main/resources/generator/android/res/values/styles.xml

```diff
@@ -1,10 +1,5 @@
 <resources>
-    <style name="GdxTheme" parent="android:Theme">
-    <item name="android:windowBackground">@android:color/transparent</item>
-        <item name="android:colorBackgroundCacheHint">@null</item>
-        <item name="android:windowAnimationStyle">@android:style/Animation</item>
-        <item name="android:windowNoTitle">true</item>
-        <item name="android:windowContentOverlay">@null</item>
-        <item name="android:windowFullscreen">true</item>
+    <style name="GdxTheme" parent="android:Theme.Holo.Light.NoActionBar.Fullscreen">
+        <item name="android:windowBackground">@color/background</item>
```

Comment:
I don't know if the defaults for Android apps will look different now that the window background is white, not transparent, but it should be easy enough for users to change back.

Reply:
I haven't checked, but we shouldn't see the window background and the transparent colour probably didn't do much anyway since `android:windowIsTranslucent` wasn't set. Personally, I think black is a safer default in case it is ever seen, e.g. if someone shrinks the game view to place an ad banner, but it's not something I'm going to get excited over.